src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -65,7 +68,7 @@ const defaultProps = {
 };
 
 const ReportActionContextMenu = (props) => {
-    const wrapperStyle = getReportActionContextMenuStyles(props.isMini);
+    const wrapperStyle = getReportActionContextMenuStyles(true);
```

Comment:
Was this just for testing or are we supposed to set this `true`?

Reply:
Oh, nice catch!